### DISCUSSION TITLE
是否考虑加入gpt-3.5-enhanced-for-role-play模型

### DISCUSSION THREAD

**Discussion by yalwolf** [timqian/openprompt.co#64] [Ideas]
gpt-3.5-enhanced-for-role-play模型是否有考虑加入

**Comment by timqian** [comment]
官方似乎没有这个模型啊
<img width="215" alt="Screenshot 2023-08-07 at 11 33 52" src="https://github.com/timqian/openprompt.co/assets/5512552/f0416fcb-4fbd-4c8d-874c-be5a11efb0bf">
